toar/toar_planetscope.ipynb

```diff
@@ -0,0 +1,474 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Converting PlanetScope Imagery from Radiance to Reflectance"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Planet Labs' analytic data products (both Rapideye and Dove) are reported in units of radiance: W/m^2. That means that every pixel in an analytic tiff has a natural language interpretation: \"How much light was captured over this spot of ground?\"\n",
```

Comment:
Using LaTeX in Jupyter notebooks: http://data-blog.udacity.com/posts/2016/10/latex-primer/